src/custom/pages/Profile/index.tsx

```diff
@@ -19,13 +19,26 @@ import useFetchProfile from 'hooks/useFetchProfile'
 import { numberFormatter } from 'utils/format'
 import useTimeAgo from 'hooks/useTimeAgo'
 import { MouseoverTooltipContent } from 'components/Tooltip'
+import NotificationBanner from '@src/custom/components/NotificationBanner'
 
 export default function Profile() {
   const referralLink = useReferralLink()
   const { account } = useActiveWeb3React()
-  const profileData = useFetchProfile()
+  const { profileData, error } = useFetchProfile()
   const lastUpdated = useTimeAgo(profileData?.lastUpdated)
 
+  const renderNotificationMessages = () => {
```

Comment:
This could improve the readability to group the other notifications, what do you think @ramirotw?

Reply:
I like the approach. I'm wondering though what happens if both banners are displayed, not sure if that's possible from a state perspective

Reply:
while I'm aware that it is currently not possible to show both statuses at the same time now (because errors are only consulted when it is mainnet). 

If you think it is better than this without grouping, let me know.